src/provider/Provider.ts

```diff
@@ -11,6 +11,11 @@ import { DDO } from '../ddo/DDO'
 
 const apiPath = '/api/v1/services'
 
+export interface urlDetails {
```

Comment:
types and interfaces should always start uppercase

Reply:
it was indented to mimic https://github.com/oceanprotocol/market/blob/main/src/utils/index.ts#L45
I'm going to update it